charts/veritable-ui/values.yaml

```diff
@@ -0,0 +1,701 @@
+  ## Please see README.md for more information
+  ## e.g:
+  ## secrets:
+  ##   - name: veritable-ui.local-tls
+  ##     key: |-
+  ##       -----BEGIN RSA PRIVATE KEY-----
+  ##       ...
+  ##       -----END RSA PRIVATE KEY-----
+  ##     certificate: |-
+  ##       -----BEGIN CERTIFICATE-----
+  ##       ...
+  ##       -----END CERTIFICATE-----
+  ##
+  secrets: []
+  ## @param ingress.extraRules Additional rules to be covered with this ingress record
+  ## ref: https://kubernetes.io/docs/concepts/services-networking/ingress/#ingress-rules
+  ## e.g:
+  ## extraRules:
+  ## - host: example.local
+  ##     http:
+  ##       path: /
+  ##       backend:
+  ##         service:
+  ##           name: example-svc
+  ##           port:
+  ##             name: http
+  ##
+  extraRules: []
+
+## @section Init Container Parameters
+##
+initDbCreate:
+  ## PostreSQL image
+  ## ref: https://hub.docker.com/_/postgres/tags/
+  ## @param initDbCreate.image.registry sqnc-routing-service image registry
+  ## @param initDbCreate.image.repository sqnc-routing-service image repository
+  ## @param initDbCreate.image.tag sqnc-routing-service image tag (immutable tags are recommended)
+  ## @param initDbCreate.image.digest sqnc-routing-service image digest in the way sha256:aa.... Please note this parameter, if set, will override the tag image tag (immutable tags are recommended)
+  ## @param initDbCreate.image.pullPolicy sqnc-routing-service image pull policy
+  ## @param initDbCreate.image.pullSecrets sqnc-routing-service image pull secrets
+  ##
+  image:
+    registry: docker.io
+    repository: postgres
+    tag: '15-alpine'
+    digest: ""
+    ## Specify a imagePullPolicy
+    ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+    ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+    ##
+    pullPolicy: IfNotPresent
+    ## Optionally specify an array of imagePullSecrets.
+    ## Secrets must be manually created in the namespace.
+    ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+    ## e.g:
+    ## pullSecrets:
+    ##   - myRegistryKeySecretName
+    ##
+    pullSecrets: []
+
+## @param initDbMigrate.enable Run database migration in an init container
+## @param initDbMigrate.environment Database configuration environment to run database into
+## @param initDbMigrate.args Argument to pass to knex to migrate the database
+initDbMigrate:
+  enable: true
+  environment: production
+  args:
+    - "migrate:latest"
+    - "--knexfile"
+    - "build/lib/db/knexfile.js"
+
+## @section Other Parameters
+##
+## @param companyHouse.url an external API from which company details can be retrieved
+## @param companyHouse.apiKey an API key
+## @param companyHouse.secret name of existing secret to use for company house API key
+
+companyHouse:
+  enabled: true
+  url: "https://api.company-information.service.gov.uk"
+  apiKey: ""
+  secret: ""
```

Comment:
Move this to just under the global parameters section and create a `## @section veritable-cloudagent configuration` replacing what is on line 63

Reply:
this should have been moved